suncityla/app/admin/action.ts

```diff
@@ -4,7 +4,7 @@ import prisma from "@/prisma/prismaClient";
 export const dynamic = "force-dynamic";
 
 const getAdmins = async () => {
-  return await prisma.admin.findMany();
+  return await prisma.user.findMany();
```

Comment:
What's the reason for changing the name from admin to user?

Reply:
So in next-auth in order to be able to connect with a provider like github or google I had to change the schema.
Please check this [link](https://authjs.dev/getting-started/adapters/prisma#schema) for reference. Since we had a model called Admin, the authentication logic didn't want to work. So I had to remove the Admin model and put it in User.